### PR TITLE
jar: add Chinese translation

### PR DESCRIPTION
**Pull request by BillLucky**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/common/jar.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# jar
+
+> Java 应用程序 / 类库打包程序。
+> 更多信息见: <https://docs.oracle.com/javase/tutorial/deployment/jar/basicsindex.html>.
+
+- 将当前目录中的所有文件递归归档到 .jar 文件中：
```
Comment (bl-ue) [inline pages.zh/common/jar.md:6]:
```suggestion
- 将当前目录中的所有文件递归归档到 `.jar` 文件中：
```

File: pages.zh/common/jar.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`jar cf {{file.jar}} *`
+
+- 将 .jar / .war 文件解压缩到当前目录：
```
Comment (bl-ue) [inline pages.zh/common/jar.md:10]:
```suggestion
- 将 `.jar` / `.war` 文件解压缩到当前目录：
```

File: pages.zh/common/jar.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`jar -xvf {{file.jar}}`
+
+- 列出 .jar / .war 文件内容：
```
Comment (bl-ue) [inline pages.zh/common/jar.md:14]:
```suggestion
- 列出 `.jar` / `.war` 文件内容：
```

File: pages.zh/common/jar.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`jar tf {{path/to/file.jar}}`
+
+- 列出带有详细输出的 .jar / .war 文件内容：
```
Comment (bl-ue) [inline pages.zh/common/jar.md:18]:
```suggestion
- 列出带有详细输出的 `.jar` / `.war` 文件内容：
```

File: pages.zh/common/jar.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# jar
+
+> Java 应用程序 / 类库打包程序。
+> 更多信息见: <https://docs.oracle.com/javase/tutorial/deployment/jar/basicsindex.html>.
```
Comment (bl-ue) [inline pages.zh/common/jar.md:4]:
```suggestion
> 更多信息见：<https://docs.oracle.com/javase/tutorial/deployment/jar/basicsindex.html>.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Comment by BillLucky** [comment]
thanks for review @bl-ue 

and I forget change master to main :)

**Review by bl-ue — APPROVED** [review]
Thank you @BillLucky :)

**Comment by bl-ue** [comment]
>and I forget change master to main :)

Oh, no problem at all.  Really.

**Review by marchersimon — APPROVED** [review]
Looks good, thanks @BillLucky

**Pull request merged**